docs/server-side/v2/legacy-api (v1)/room.mdx

```diff
@@ -11,7 +11,7 @@ You can create room using API. This provides a more scalable way of creating roo
 
 <PostRequest
     title="Create Room API"
-    desp="This endpoint is used to create Room. "
+    desp="This endpoint is used to create room. "
```

Comment:
this only changes in the legacy API docs?

Reply:
the casing was incorrect in legacy docs, hence updated it. But the changes are for the latest API docs. 
CC: @iamajunravi

Reply:
they are not updated in this PR?

Reply:
my bad - i thought room names were supporting spaces. but we have updated on template names

Reply:
👍